### PR TITLE
Fix formatting for `event.duration`

### PR DESCRIPTION
**Pull request by webmat**
Human readable not working well for us.

I don't think this is a blocker for 7.0.0. The workaround is simply to edit the Kibana index pattern's formatting options for event.duration.

But this should absolutely go to 7.0.1 if possible.

Note that this is a quick fix applied on a file generated by ECS. If we go forward with this fix, I will backport it to ECS

### CODE REVIEW

**Comment by webmat** [comment]
Here's a visualization from the Packetbeat Dashboard prior to this PR:

before-11675

Here's the same visualization after this PR:

after-11675

I don't think it looks great, but it's much more useful than the current state of things.

**Comment by webmat** [comment]
Note that I've built one of the Beats, performed its `beat setup`, ingested data and confirmed that the format takes effect properly.

**Comment by ruflin** [comment]
The new graph is better then the old one but still not very nice. Also it lacks the unit.

I think this is a more general issue Kibana https://github.com/elastic/kibana/issues/20374 

I'm good with getting this into Beats for now but not sure if I would make it part of ECS directly as I would hope we could get overall a better solution long term.

At the same time I would not consider these changes to be breaking changes so we could add it to ECS and change it later.

**Comment by ruflin** [comment]
jenkins, test this

File: libbeat/_meta/fields.ecs.yml [hunk 1]
```diff
@@ -613,6 +613,8 @@
       type: long
       format: duration
       input_format: nanoseconds
+      output_format: asMilliseconds
+      output_precision: 1
```
Comment (simitt) [inline libbeat/_meta/fields.ecs.yml]: Just as a side note, we previously had some confusion around showing durations that are stored and declared as nanoseconds in a different format (as this also changes format in discover), see https://github.com/elastic/apm-server/pull/1717

**Pull request merged**